pkgs/applications/misc/crumbs/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, automake, autoconf, gzip, ruby, bash }:
```

```suggestion
{ stdenv, fetchFromGitHub, autoreconfHook, gzip, ruby, bash }:
```
This is the reason of the ofborg error. You can get access to the error message by running `nix-review pr 58960 -p crumbs`:
```
error: while evaluating 'callPackageWith' at /home/symphorien/.cache/nix-review/pr-58960-1/nixpkgs/lib/customisation.nix:108:35, called from /home/symphorien/.cache/nix-review/pr-58960-1/nixpkgs/pkgs/top-level/all-packages.nix:699:12:
while evaluating 'makeOverridable' at /home/symphorien/.cache/nix-review/pr-58960-1/nixpkgs/lib/customisation.nix:67:24, called from /home/symphorien/.cache/nix-review/pr-58960-1/nixpkgs/lib/customisation.nix:112:8:
undefined variable 'autoreconfHook' at /home/symphorien/.cache/nix-review/pr-58960-1/nixpkgs/pkgs/applications/misc/crumbs/default.nix:15:7
```

---

I didn't know about that tool, thanks